graylog2-web-interface/src/components/visualizations/QuickValuesVisualization.jsx

```diff
@@ -192,7 +194,10 @@ const QuickValuesVisualization = React.createClass({
           // different terms in the stacked value more visible.
           formattedTerm = this.props.data.terms_mapping[d.term].map(t => t.value).join(' <strong style="color: #999999;">&mdash;</strong> ');
         }
-
+        if (this.props.field === 'timestamp') {
+          // convert unix timestamp to proper formatted value, so that add to search button works correctly
+          formattedTerm = moment.utc(Number(d.term)).format(DateTime.Formats.TIMESTAMP);
```

Comment:
This does not display date times in the user time zone, providing inconsistent values with the search results. We should use `new DateTime(Number(d.term)).toString(DateTime.Formats.TIMESTAMP)` to get the time formatted in the user's time zone.